JustSaying.IntegrationTests/Fluent/Subscribing/WhenHandlerIsDeclaredAsExactlyOnce.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using JustSaying.IntegrationTests.TestHandlers;
+using JustSaying.Messaging;
+using JustSaying.Messaging.MessageHandling;
+using JustSaying.TestingFramework;
+using Microsoft.Extensions.DependencyInjection;
+using Shouldly;
+using Xunit.Abstractions;
+
+namespace JustSaying.IntegrationTests.Fluent.Subscribing
+{
+    public class WhenHandlerIsDeclaredAsExactlyOnce : IntegrationTestBase
+    {
+        public WhenHandlerIsDeclaredAsExactlyOnce(ITestOutputHelper outputHelper)
+            : base(outputHelper)
+        {
+        }
+
+        [AwsFact]
+        public async Task Then_Both_Handlers_Receive_The_Message()
```

Comment:
Is this method correctly named? IFrom the code it seems more like "then the handler is called once"

Reply:
Copy-pasta.